### ISSUE TITLE
Obtain home addresses of campaign participants

### ISSUE THREAD

**Issue by d70-t** [halo-db/storymap#49]
As a climate change skeptic, I want to obtain a list of home addresses of campaign participants, ideally grouped by the participants function during the campaign, such that it is simpler to write threatening letters or motivate other skeptics to do what they need to do.

**Comment by rico-hengst** [comment]
You are right. Keep in mind, that this use case is in contrast to the (reutation) wish of the author/scientist to publish datasets via DOI, see https://github.com/halo-db/storymap/issues/42 .
